samples/grpc/local-drone-control-scala/src/main/resources/cluster.conf

```diff
@@ -1,5 +1,3 @@
 akka {
   actor.provider = cluster
```

Comment:
maybe this config file is just confusing for this sample, and we can put this in application.conf and application-kubernetes.conf ?

Reply:
We could put more cluster config here, for the clustered variants, and then only have `provider = cluster` in the default single-node `application.conf`.

Reply:
ok, I think something is missing for the single node one as well. contact-point-discovery config should be there. That is now only in local-shared.conf. The local files are intended for local dev mode.

Reply:
Single-node has its own main, does a self-join there, and doesn't start cluster bootstrap. Local has config discovery.

Reply:
alright, I'll look over the stuff. confusing to use different approaches, but nothing you should spend time on in this PR